src/components/messages/RatingMessage.vue

```diff
@@ -44,38 +44,37 @@ export default {
         comment: '',
         success: false
       },
-      stars: 5, // Total number of stars
-      selectedStar: 0, // Currently selected star
-      hoveredStar: -1, // Currently hovered star
+      selectedRatingItem: 0, // Currently selected Item
+      hoveredRatingItem: -1, // Currently hovered Item
       disabled: false // Flag to disable the component
     };
   },
   props: {
     message: {
       type: Object,
       required: true,
-      validator: (message) => {
-        return (
-          message &&
-          message.type === 'rating' &&
-          message.data &&
-          message.data.title &&
-          message.data.maxValue &&
-          message.data.commentsAllowed
-          /* Other elements that may have the rating message */
-        );
-      }
+      validator: function(message) {
+          const hasType = typeof message.type === 'string' && message.type === 'rating';
+          const hasTitle = typeof message.data.title === 'string' && message.data.title.trim().length > 0;
+          const hasMaxValue = typeof message.data.maxValue === 'number' && message.data.maxValue > 0;
+          const hasCommentsAllowed = typeof message.data.commentsAllowed === 'boolean';
+          return hasType && hasTitle && hasMaxValue && hasCommentsAllowed;
+        }
     }
   },
   computed: {
     ratingText() {
       return this.message.data.title;
-    }
+    },
+
+    maxRating() {      
+      return this.message.data.maxValue || 0; 
```

Comment:
I wonder whether we should not put default 5 - for backwards compatibility?
Then if people have been passing it without a `maxValue` (and ignoring the resulting warnings in the console) - it will still behave the same way